server/public/model/config.go

```diff
@@ -3684,6 +3685,11 @@ func (o *Config) isUpdate() bool {
 func (o *Config) SetDefaults() {
 	isUpdate := o.isUpdate()
 
+	if o.DisplaySettings.SuggestionDebounceDelay == nil {
+		defaultDelay := 100
```

Comment:
please create a constant for default delay in the file where other defaults are defined

Reply:
Which file is that, please?